.gitignore

```diff
@@ -1,3 +1,3 @@
 .pio
 .vscode
-*_env.ini
+*_env.ini
```

Comment:
Could you restore the original file, this way this file will not be identified in this PR

Reply:
How would I do that? I feel like it would involve a git command.

Reply:
No, if you take the original file and replace the current one by the original, commit and push it. It should not appear anymore here.

Reply:
I figured it out by `git restore` from master before I saw your simpler answer